testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/InfinispanStatistics.java

```diff
@@ -82,7 +82,7 @@
     void reset();
 
     /**
-     * Returns {@code true} iff the statistics represented by this object can be retrieved from the server.
+     * Returns {@code true} if the statistics represented by this object can be retrieved from the server.
```

Comment:
iff is not a typo, it means "if and only if": https://en.wikipedia.org/wiki/If_and_only_if